### ISSUE TITLE
Vote down automatically generated chapters

### ISSUE THREAD

**Issue by melissis** [ajayyy/SponsorBlock#1616]
I suggest there be a function to "vote down" YouTube's automatically generated chapters (i.e. to hide them). Right now it's a mess when users and YouTube's AI have different ideas of what the chapters should be.

Might as well make it possible to vote down channel-made chapters while we're at it.

**Comment by ajayyy** [comment]
Yea, they are getting very common now

**Comment by melissis** [comment]
`Vote down non-YouTube chapters`
Did you mean "non-SB chapters"?

**Comment by mirror176** [comment]
I thought youtube chapters were something the channel creator is manually tagging for their video. If youtube does it automatically, is there a way to tell ones that came from creator vs an AI marking them?

**Comment by ajayyy** [comment]
@mirror176 not in a good way that I know of, right now there is a message but it changes per language

**Comment by melissis** [comment]
> is there a way to tell ones that came from creator vs an AI marking them?

Yes. I think ajayyy meant programmatically, but from a user perspective, it's easy. If the channel has made the chapters, there are timestamps in the description (unless there is another way to do it now). And if you click "View all" under "Chapters" (after opening the full description), it says "These chapters are auto-generated". And also, you can often tell just by noticing that something is off.
